packages/sync/src/utils/manageSnapshotTable.ts

```diff
@@ -0,0 +1,81 @@
+import { TupaiaDatabase } from '@tupaia/database';
+
+const SCHEMA = 'sync_snapshots';
+
+const assertIfSessionIdIsSafe = (sessionId: string) => {
+  const safeIdRegex = /^[A-Za-z0-9-]+$/;
```

Comment:
Will the length of this be known?

```suggestion
  const safeIdRegex = /^[A-Za-z0-9-]{24}$/;
```